### PR TITLE
Add portfolio examples

### PR DESCRIPTION
**Pull request by paked**
I'm not quite sure this should be our final copy. Do we have
permission to use these?

I would much rather have links to the interesting sites built by
our hackers, as opposed to the ones which followed the rules

### CODE REVIEW

**Comment by jonleung** [comment]
I think this would be great to have in our final copy! However, I'm not sure who did give / didn't yet give permission for us to link to their website.

Here is a checklist of of the websites and who has given us permission so far:

# Example Websites

Here are some examples websites other hackers made:

## Basic
 
- [ ] <a href="https://output.jsbin.com/fetice" target="_blank">Doge</a>
- [ ] <a href="https://output.jsbin.com/nocotu" target="_blank">Anti-Internet</a>
- [ ] <a href="https://output.jsbin.com/wopikas" target="_blank">Miku</a>
- [ ] <a href="https://output.jsbin.com/zilola" target="_blank">Leia</a>
- [ ] <a href="https://output.jsbin.com/muwaha" target="_blank">I don't know you</a>
- [ ] <a href="https://harrison.tech/zero-cool" target="_blank">Zero Cool</a>

## Advanced
 
- [x] <a href="https://cydrobolt.com/" target="_blank">Chaoyi Zha</a>
- [ ] <a href="http://jevinsidhu.github.io/" target="_blank">Jevin Sidhu</a>

**Comment by jonleung** [comment]
For websites that are _not_ JS Bin, I think it's fine to just link them (such as jonl.org). However, for the websites hosted on JS Bin websites, because they are kinda private because you have to have the URL in order to be able to view them, I think we should ask for their permission. We may have to text Hack Series participants in order to get their permission...

**Comment by saescapa** [comment]
How about making a bunch at a hackNight? It would interesting and fun.

**Comment by MaxWofford** [comment]
@saescapa :+1:

**Comment by zachlatta** [comment]
What if we held a Hack Night tonight (Friday) and made the theme "make the
craziest portfolio website". We could all demo at the end of it and add the
best ones as examples to the workshop itself.



On Fri, Jan 1, 2016 at 1:41 AM, Max Wofford <notifications@github.com>
wrote:

> @saescapa <https://github.com/saescapa> [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/610#issuecomment-168295619>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Comment by paked** [comment]
@zachlatta, sounds good to me!

**Comment by jonleung** [comment]
I think it may actually be worthwhile to actually ask the hackers for their permission to link to their websites as well because then we can say these are "real" websites that other people made. I don't have bandwidth right now to commit to doing that right now.

Have fun at the Hack Night!

**Comment by jonleung** [comment]
Going to assign this back to you @paked until this is ready to merge!

**Comment by paked** [comment]
No problem.

**Comment by zachlatta** [comment]
> I think it may actually be worthwhile to actually ask the hackers for
their permission to link to their websites as well because then we can say
these are "real" websites that other people made. I don't have bandwidth
right now to commit to doing that right now.

How would asking people for permission to link to their websites make them
"real"? Aren't they real websites that other people made regardless?

On Fri, Jan 1, 2016 at 3:10 PM, Harrison Shoebridge <
notifications@github.com> wrote:

> No problem.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/610#issuecomment-168349153>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Comment by jonleung** [comment]
I mean its the difference between "other real hackeres made this website" vs "your teachers made these real websites"

**Comment by zachlatta** [comment]
Aren't we talking about linking to other "real hackers'" websites though? I
thought this discussion was about the ethics of not asking before linking.

On Fri, Jan 1, 2016 at 3:29 PM, Jonathan Leung <notifications@github.com>
wrote:

> I mean its the difference between "other real hackeres made this website"
> vs "your teachers made these real websites"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/610#issuecomment-168350366>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

File: workshops/portfolio/README.md [hunk 1]
```diff
@@ -555,6 +555,17 @@ Huzzah, we did it! You can take this URL and share it with your friends now!
 One last thing. Please click a rating below to rate this workshop. It'll only
 take 3 seconds.
 
+## Continuation
+
+Want to continue building? I challenge you to make your website *even more
+cool!
```
Comment (MaxWofford) [inline workshops/portfolio/README.md]: It looks like the asterisk may have lost it's closing.

**Comment by MaxWofford** [comment]
> "other real hackers made this website" vs "your teachers made these real websites"

@jonleung This was true for Hack Series, but I don't think we are considered teachers for hackers in our clubs. It seems like our content is showing how we learned coding through projects (I personally learned HTML & CSS through making sites like these), so I think it is acceptable to include our own projects.

File: workshops/portfolio/README.md (hunk 1 above)
Comment (zachlatta) [inline workshops/portfolio/README.md]: Shouldn't this also use underscores instead of asterisks for emphasis?

**Comment by jonleung** [comment]
Ok! I'm comfortable with showing our own work. I'm comfortable with merging this pull request in once this commentchttps://github.com/hackclub/hackclub/pull/610#discussion_r49284585 has been addressed.

**Comment by paked** [comment]
@jonleung do we still need this, perhaps it would be better to do this while rewriting the portfolio workshop?

**Comment by zachlatta** [comment]
:+1: I think it'd be best to include this while rewriting the Portfolio
workshop

On Sun, Jan 17, 2016 at 10:16 PM, Harrison Shoebridge <
notifications@github.com> wrote:

> @jonleung <https://github.com/jonleung> do we still need this, perhaps it
> would be better to do this while rewriting the portfolio workshop?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/610#issuecomment-172436930>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

**Comment by jonleung** [comment]
Cool, I will add some of these to our portfolio website example. I will close this PR for now.

**Pull request closed without merging**